libnavigation-ui/src/main/res/layout-land/instruction_content_layout.xml

```diff
@@ -28,8 +28,8 @@
             android:layout_width="40dp"
             android:layout_height="40dp"
             android:layout_margin="8dp"
-            app:maneuverViewPrimaryColor="?attr/navigationViewBannerManeuverPrimary"
-            app:maneuverViewSecondaryColor="?attr/navigationViewBannerManeuverSecondary"/>
+            app:maneuverViewPrimaryColor="@color/mapbox_instruction_maneuver_view_primary"
```

Comment:
Unlike others why did we add the colors here. I see that the colors are being also added programatically.

Reply:
this is just for preview purpose. the color is still set programmatically. I could remove it

Reply:
Yeah I think let's remove it to keep the parity with other xml's